homeassistant/components/mqtt_eventstream.py

```diff
@@ -20,19 +20,27 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.remote import JSONEncoder
 
+import homeassistant.const as has_const
+
 DOMAIN = 'mqtt_eventstream'
 DEPENDENCIES = ['mqtt']
 
 CONF_PUBLISH_TOPIC = 'publish_topic'
 CONF_SUBSCRIBE_TOPIC = 'subscribe_topic'
 CONF_PUBLISH_EVENTSTREAM_RECEIVED = 'publish_eventstream_received'
+CONF_IGNORE_EVENT = 'ignore_event'
+
+EVENT_TYPES = [has_const.__dict__[x]
```

Comment:
Riiiiiight 😉 

I don't think that we have to limit it to HASS events. What if someone uses a custom event and doesn't want it to be published?

Reply:
gotcha. i wasn't aware custom events were a thing! will update.